index.html

```diff
@@ -0,0 +1,144 @@
+                <h2 class="venue">@ PleasurePark, Airforce PortHarcourt </h2>
+            </section2>
+        </div>
+        </section2>
+    </header>
+
+    <!-- main section -->
+    <main>
+        <section class="main-programs">
+            <h2>Main Program</h2>
+            <hr>
+            <div class="progg">
+                <div class="prog">
+                    <img src="./img/prog1.png" alt="program icon">
+                    <h3>Lecture </h3>
+                    <p> Meet the Keynote Speakers as they take turns to speak on pushing for a better world where we can
+                        continue to eat, play and connect. </p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog2.png" alt="program icon">
+                    <h3>Exhibitions</h3>
+                    <p> Inspections of an array of Bole and perculiar to various regions where chefs would give
+                        illustrate
+                        talks on how best their bole is served </p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog3.png" alt="program icon">
+                    <h3>Forum </h3>
+                    <p> chefs, stakeholders and participants interactive sessions, opinions and ideas would b e
+                        exchanged.
+                    </p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog4.png" alt="program icon">
+                    <h3>merchandise </h3>
+                    <p> Get the latest street Tech innovations.We have put together a Limited merch collections.</p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog5.png" alt="program icon">
+                    <h3>workshop</h3>
+                    <p> Introduction to new technological advancements in the processing of our local Bole</p>
+                </div>
+            </div>
+            <div class="join-bole">
+                <h4> Join Bole Festival 2033</h4>
+            </div>
+        </section>
+        <section class="featured-chefs">
+            <h2> Bole Festival Chefs</h2>
+            <hr>
+            <div class="chefz">
+            </div>
+            <button class="morebtn">
+                <h4>More <span><i class="ri-arrow-drop-down-line"></i></span></h4>
+            </button>
+        </section>
+    </main>
+    <footer>
+        <section class="partners">
+            <h4>
+                Sponsored By
+            </h4>
+            <hr>
+
+            <div class="partners-logo" id="sponsors">
+                <img src="./img/GTBank_logo.png" alt="gtbank">
+                <img src="./img/coker.png" alt="cocacola">
+                <img src="./img/Government_of_Rivers_State_logo.png" alt="Rs-govt">
+                <img src="./img/Rhythm_937_FM.png" alt="93.7fm">
+                <img src="./img/McDonald's_logo.png" alt="McDonald">
```

Comment:
- [ ]  Featured chief's image should be at the left side of the texts. make sure that you have an image at the left and the text section at the right of it and  fill the page by using proper font size, so that you can match the provided behance design
![image](https://user-images.githubusercontent.com/43877689/227661260-6624d021-f15b-4ca9-90eb-532e4d0074f5.png)